lib/bundle/brew_installer.rb

```diff
@@ -236,7 +236,8 @@ def resolve_conflicts!(verbose:)
     end
 
     def install!(verbose:)
-      puts "Installing #{@name} formula. It is not currently installed." if verbose
+      with_args = @args.empty? ? "" : " with args #{@args}"
```

Comment:
```suggestion
      with_args = " with args #{@args}" if @args.present?
```

as it'll evaluate to `nil` otherwise which outputs the same when interpolated